ext/gd/libgd/gd_interpolation.c

```diff
@@ -62,7 +62,7 @@
 #include "gdhelpers.h"
 #include "gd_intern.h"
 
-#ifdef _MSC_VER
+#if defined(_MSC_VER) && !defined(__clang__)
```

Comment:
This is a stop-gap measure to avoid yet another warning. I believe this code makes no sense, and emmintrin.h is not even required here, but that needs to be resolved upstream first.

Reply:
Sigh: https://github.com/libgd/libgd/pull/945

Reply:
As the PR got merged upstream, can you just remove it now?

Reply:
Ah, right. Just did that.